.prettierrc

```diff
@@ -2,5 +2,6 @@
 	"semi": true,
 	"singleQuote": false,
 	"tabWidth": 2,
-	"trailingComma": "es5"
+	"trailingComma": "es5",
+	"plugins": ["prettier-plugin-tailwindcss"]
 }
```

Comment:
Maybe delete `prettier.config.js` in conjunction with this change (where the plugin was defined before): 
https://prettier.io/docs/en/configuration.html

Reply:
Done